### ISSUE TITLE
Is it possible for one to create and href in a page and when clicked upon goes to the next page of directry.

### ISSUE THREAD

**Issue by gibertgibs** [gibertgibs/Self-Motive4Tech#4]
Page 1 | Page 2 | Page 3
---------------------------------------
Click    | Press   | Smile
-------------------------------

Is it possible for me to open click to lead me to press and press to smile. Is this possible.

**Comment by timmwille** [comment]
not sure if I get you right, so you want to switch from one page to another?
Or just klick "next" like in a gallery?

Here is a nice page, so yes, everything is possible, not sure if I can help you out though on how to make it happen :)
https://sachinchoolur.github.io/lightGallery/

**Comment by timmwille** [comment]
https://github.com/collections/github-pages-examples

**Comment by bmen** [comment]
Output:
Page 1 | Page 2 | Page 3
------------ | ------------- | -------------
Click | Press | Smile

Code:
```
Page 1 | Page 2 | Page 3
------------ | ------------- | -------------
Click | Press | Smile
```

**Comment by bmen** [comment]
The lines
```
------------ | ------------- | -------------
```
after the headline row are important

The following rows do not need the lines anymore!
```
Page 1 | Page 2 | Page 3
------------ | ------------- | -------------
Click | Press | Smile
Click | Press | Smile
```

**Comment by bmen** [comment]
> Is it possible for me to open click to lead me to press and press to smile. Is this possible.

You can also insert a link in a table.I don't know if you meant that.

Code:
```
Page 1 | Page 2 | Page 3
------------ | ------------- | -------------
Click | Press | [Smile](https://fontawesome.com/icons/smile?style=regular)
Click | Press | Smile
```
Output:
Page 1 | Page 2 | Page 3
------------ | ------------- | -------------
Click | Press | [Smile](https://fontawesome.com/icons/smile?style=regular)
Click | Press | Smile

**Comment by gibertgibs** [comment]
Exactly that's what i meant but then I want it to lead me to one of my project pages like click on it from Readme file but then leads me to the details.

**Comment by bmen** [comment]
The details are then on another page or in the readme file? It is also possible to link to a headline. Then the browser scrolls to this position.

**Comment by gibertgibs** [comment]
Yes that's exactly what I want to do but it's shouldn't lead me to any website like this type of linking [ASKnet](http://ASKnet.....) not this

**Comment by timmwille** [comment]
so you want to create a menu @gibertgibs ?

**Comment by gibertgibs** [comment]
Yes you got me write.

**Comment by timmwille** [comment]
maybe @bmen can help with this

**Comment by gibertgibs** [comment]
Okay fine then.
